pkg/auth/authproviders/oidc/backend_impl.go

```diff
@@ -562,12 +565,19 @@ func (p *backendImpl) Validate(context.Context, *tokens.Claims) error {
 // Helpers
 ///////////
 
-// UserInfo is an internal helper struct to unmarshal OIDC token info into.
+// userInfoType is an internal helper struct to unmarshal OIDC token info into.
 type userInfoType struct {
 	Name   string   `json:"name"`
 	EMail  string   `json:"email"`
 	UID    string   `json:"sub"`
 	Groups []string `json:"groups"`
+	// Claim "account_id" is the claim that represents organisation id within sso.redhat.com.
+	// Red Hat users are united in organisations.
+	// See more on the claims here:
+	// https://source.redhat.com/groups/public/it-user/it_user_team_wiki/topic_external_sso_enablements#attributes-needed
+	OrgID string `json:"account_id"`
+	// Claim "is_org_admin" is the claim that identifies organisation admins within sso.redhat.com.
```

Comment:
I'm sure you did a manual test of that against our stage RH SSO, could you please update the screenshot in the *Testing Performed* section?